frontend/src/components/layout/Navigation.tsx

```diff
@@ -23,7 +22,7 @@ export default function Navigation() {
                 <li><Link to={"/"}>Home</Link></li>
                 <li><Link to={"/recipes"}>Recipes</Link></li>
                 <li className={"icon-place button"}>
-                    <Button onClick={handleClick} startIcon={<PostAdd/>}>Add Recipe</Button>
+                    <a href="#" onClick={handleClick}>Add Recipe</a>
```

Comment:
in a react context a href to an # is not allowed. An a-Tag should lead to a new page. Instead you could use a button here :-)

Reply:
I want to change the whole navigation bar in the next couple of day to mui nice Bar. I kept a-Tag for now, but I removed the href. I hope, it's okey now?

Reply:
now I have a readability issue without the href. :'( I put it back for now. It will be adjusted in the next days for sure! I hope it's okey for you?